build/__extVarNames.sh

```diff
@@ -5,6 +5,16 @@ APPLICATION_INSIGHTS_INSTRUMENTATION_KEY_ENV_VAR_NAME='ORYX_AI_INSTRUMENTATION_K
 APP_SERVICE_APP_NAME_ENV_VAR_NAME='APPSETTING_WEBSITE_SITE_NAME'
 SCM_COMMIT_ID_ENV_VAR_NAME='SCM_COMMIT_ID'
 PHP_ORIGIN='PHP_ORIGIN'
+PHP_FPM_CONFIGURATION_FILE='/usr/local/etc/php-fpm.d/www.conf'
```

Comment:
I added these to the `constants.yaml` file, since these variables were getting overwritten in other files when the `generateConstants.sh` script was run.